src/assets.rs

```diff
@@ -15,7 +15,7 @@ use crate::syntax_mapping::ignored_suffixes::IgnoredSuffixes;
 use crate::syntax_mapping::MappingTarget;
 use crate::{bat_warning, SyntaxMapping};
 
-use lazy_theme_set::LazyThemeSet;
+pub use lazy_theme_set::LazyThemeSet;
```

Comment:
Spontaneously I would like to have a public API that hides wether or not themes are lazy-loaded. So exporting the name `LazyThemeSet` should be avoided if possible.

I'm a bit short on time, so maybe you could experiment a bit and see if that can be achieved?